skills/research/SKILL.md

```
@@ -0,0 +1,404 @@
+*Sources: {sourcesGathered} | Searches: {totalSearches} | Subagents: {subagentCalls} | Iterations: {iteration} | Duration: {duration} | Date: {date}*
+```
+
+Set `phase="DONE"`.
+
+On completion: The Stop hook will display a resource usage summary when you exit.
+</phase>
+
+</workflow>
+
+<error_handling>
+| Error | Action |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)